scripts/Fig3D_merged-umaps.R

```diff
@@ -0,0 +1,90 @@
+# This script is used to generate a small version of the merged UMAP for Figure 3
+
+# load project
+renv::load()
+
+library(SingleCellExperiment)
+library(dplyr)
+library(ggplot2)
+
+# Set default ggplot theme
+theme_set(
+  theme_classic() +
+    theme(
+      #plot.margin = margin(rep(20, 4)),
+      strip.background = element_rect(fill = "transparent", linewidth = 0.5),
+      # no axis ticks or labels
+      axis.line = element_blank(),
+      axis.ticks = element_blank(),
+      axis.text = element_blank(),
+      # add a square around each of the plots
+      panel.background = element_rect(colour = "black", linewidth=0.5),
+      aspect.ratio = 1
+    )
+)
+
+set.seed(2024)
+
+# Set up -----------------------------------------------------------------------
+
+# folder where any local data lives
+local_results_dir <- here::here("s3_files", "SCPCP000003")
+
+# define file path to merged object 
+merged_sce_file <- file.path(local_results_dir, "SCPCP000003_merged.rds")
+
+# read in merged sce objects 
+merged_sce <- readr::read_rds(merged_sce_file)
+
+# define output file paths 
+png_dir <- here::here("figures", "pngs") 
+output_png_file <- file.path(png_dir, "Fig3D_merged-umaps.png")
+
+pdf_dir <- here::here("figures", "pdfs") 
+output_pdf_file <- file.path(pdf_dir, "Fig3D_merged-umaps.pdf")
+
+
+# Prep for plotting ------------------------------------------------------------
+
+libraries_to_plot <- c("SCPCL000050",
```

Comment:
I wonder if this should be in the `setup` portion of the script, since I imagine this might change later.

Reply:
I think my plan is to not change this later. I don't think it matters which libraries we use, I just want to make sure that if the merged object gets run on more libraries, we are always picking the same 4.